packages/amplify-category-auth/resources/cloudformation-templates/auth-template.yml.ejs

```diff
@@ -319,6 +321,7 @@ Resources:
       GenerateSecret: !Ref userpoolClientGenerateSecret
       RefreshTokenValidity: !Ref userpoolClientRefreshTokenValidity
       UserPoolId: !Ref UserPool
+      PreventUserExistenceErrors: ENABLED
```

Comment:
I don't think this is your change. Can you rebase?

Reply:
No it's not

Reply:
I rebase there seems to be a bad merge I can checkout re commit